integrations/phoneinfoga/compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
```

Comment:
use version 3.8 if you can, like we do in all other docker compose files in the project